src/lib/utils/geoMath.ts

```diff
@@ -0,0 +1,33 @@
+import mapboxgl from 'mapbox-gl';
+
+const rad2degr = (rad: number): number => (rad * 180) / Math.PI;
+
+const degr2rad = (degr: number): number =>(degr * Math.PI) / 180;
+
+export const getLatLngCenter = (latLngInDegr: Array<mapboxgl.LngLatLike>): mapboxgl.LngLatLike => {
+  const LATIDX: number = 0;
+  const LNGIDX: number = 1;
+  let sumX: number = 0;
+  let sumY: number = 0;
+  let sumZ: number = 0;
+
+  for (let i = 0; i < latLngInDegr.length; i += 1) {
+    const lat: number = degr2rad(latLngInDegr[i][LATIDX]);
+    const lng: number = degr2rad(latLngInDegr[i][LNGIDX]);
```

Comment:
`ERROR in [at-loader] ./src/lib/utils/geoMath.ts:16:34 
    TS7053: Element implicitly has an 'any' type because expression of type 'number' can't be used to index type 'LngLatLike'.
  No index signature with a parameter of type 'number' was found on type 'LngLatLike'.`

Reply:
Vyřešeno konverzí souřadnic, viz `mapboxgl.LngLat.convert(coords)`